testing/molecule/default/search-descriptions.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Search with & without -l flag
+  hosts: all
+  become: false
+  vars:
+    package: make
+
+  tasks:
+    - name: Search with -l
+      raw: "nupac search {{ package }} --long | to json"
+      args:
+        executable: "{{ nu }}"
+      register: search-l
+
+    - name: Assert that long-desc was used
+      assert:
+        that:
+          - "(search-l.stdout | from_json | json_query('description')) == 'This module contains completions for make build tool
+          '"
+
+    - name: Search without -l
+      raw: "nupac search {{ package }} | to json"
+      args:
+        executable: "{{ nu }}"
+      register: search
+
+    - name: Assert that short-desc was used
+      assert:
+        that:
```

Comment:
that: "search.stdout | from_json | json_query('description') == 'makefile completions for nu'"